docs/plugins/ryd.md

```diff
@@ -0,0 +1,18 @@
+# Return YouTube Dislike
+
+## What is Return YouTube Dislike?
+Return YouTube Dislike (or RYD) is an third party API that returns dislike counts for YouTube videos after they got removed in December 2021. It's avaiable in **VueTube** as a plugin. Dislike stats aren't exact but they are very acurrate, using a combination of archieved dislike counts of old videos, votes by users and extrapolation. In the future creators will be able to share their own dislike counts from Youtube Studio, according to their [documentation](https://returnyoutubedislike.com/faq).
```

Comment:
- is **a** third-party API
- available instead of avaiable 
- accurate instead of acurrate
- archived instead of archieved